### PR TITLE
[ALLUXIO-1833] Consolidate normalizePath method in S3UnderFileSystem

### PR DESCRIPTION
**Pull request by ifcharming**
https://alluxio.atlassian.net/browse/ALLUXIO-1833

### CODE REVIEW

File: underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java [hunk 1]
```diff
@@ -591,4 +590,14 @@ private String stripPrefixIfPresent(String key) {
...
     return key;
   }
+
+  /**
+   * Adds a trailing {@link S3UnderFileSystem#PATH_SEPARATOR} if it does not exists in path.
+   *
+   * @param path the file name
+   * @return updated path with trailing {@link S3UnderFileSystem#PATH_SEPARATOR}
+   */
+  private String normalizePath(String path) {
```
Comment (apc999) [inline underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java]: how about moving this to PathUtil?

**Comment by apc999** [comment]
One comment left. Other looks good to me

File: underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java (hunk 1 above)
Comment (ifcharming) [inline underfs/s3/src/main/java/alluxio/underfs/s3/S3UnderFileSystem.java]: Good point. Moving this to PathUtils also benefits other UFS. Updated.

**Comment by ifcharming** [comment]
Thanks @apc999 , PR updated. PTAL

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**